### DISCUSSION TITLE
Please, help me..  (bitcoinlib.wallets.WalletError: Create transaction: No unspent transaction outputs found or no key available for UTXO's)

### DISCUSSION THREAD

**Discussion by kernaldpark** [1200wd/bitcoinlib#375] [General]
1. I created a new wallet with a new private key (cTWsDP24LffAVFPhgC7Hmx4aC8nJbVC6**************) - Success
 2. And I received some Bitcoin from faucet site - Success
 3. An error occurred during the process of transferring the received Bitcoin to another address. If someone knows any solution for anyone of them, I will be thankful for your help
 
**keys = 'cTWsDP24LffAVFPhgC7Hmx4aC8nJbVC6**************'
w = wallet_create_or_open(name='today240303wallet1901', keys=keys, network='testnet')
w.scan(scan_gap_limit=1)
w.info(detail=5)**

"""
=== WALLET ===
 ID                             1
 Name                           today240303wallet1901
 Owner
 Scheme                         bip32
 Multisig                       False
 Witness type                   legacy
 Main network                   testnet
 Latest update                  2024-03-03 19:17:57.457662

= Wallet Master Key =
 ID                             1
 Private                        True
 Depth                          0

- NETWORK: testnet -
- - Keys
    1 m                            n4ffehNw8k7vfzQvaCokiFWJZ2aSDab7RH            today240303wallet1901               0.00000000 TBTC
    2 m/44'                        mjjQYSug8FApk74pFhxKqm2PouRYswUEeb            purpose 44                          0.00000000 TBTC
    3 m/44'/1'                     n1owSexbbJNZo1Pme76m4zz2FwYepfHBZZ            coin type 1                         0.00000000 TBTC
    4 m/44'/1'/0'                  my9pNaA8VoXWiPDKpMQoTp9WZuMZNZUKRZ            account 0                           0.00000000 TBTC
    5 m/44'/1'/0'/0                moX1drTBjNzX2MkQJbjthWeWKETairJRmM            change 0                            0.00000000 TBTC
    7 m/44'/1'/0'/1                mjtLaa9DxTdNAkZbQqi8TvUXmyqmKcberM            change 1                            0.00000000 TBTC
    **6 m/44'/1'/0'/0/0              mmK8PcMau3DMfdtJzqqQG1HZnVaWyYvApE            address index 0                     0.00001000 TBTC**
    8 m/44'/1'/0'/1/0              n3hPnBuSUWAfTRzgWD2G7Qic94C7CUkkMj            address index 0                     0.00000000 TBTC
    9 m/44'/1'/0'/0/1              n45mnqXAWsnXLJ7uGUoy4ZHnU7JG26fPZb            address index 1                     0.00000000 TBTC

- - Transactions Account 0 (1)
39073dfde2ded80f8e56da8404f59f6c68e94e9a336534a9bb4282e312cb52a8          mmK8PcMau3DMfdtJzqqQG1HZnVaWyYvApE        0       0.00001000 TBTC U

= Balance Totals (includes unconfirmed) =
testnet              (Account 0)               0.00001000 TBTC
"""

**tx_fee = 250
receiver_address = 'tb1q6v2tn4k7j9w5xxvwepkhdjrdpuwlwjcw3par5r'
t = w.send_to(receiver_address, 750, min_confirms=0, fee=tx_fee)
t.info()**

....
....
  File "D:\GIT\Python\bitcoin\.venv\lib\site-packages\bitcoinlib\wallets.py", line 3424, in select_inputs
    raise WalletError("Create transaction: No unspent transaction outputs found or no key available for UTXO's")
bitcoinlib.wallets.WalletError: Create transaction: No unspent transaction outputs found or no key available for UTXO's

**Comment by kernaldpark** [comment]
1. Each account address must hold a minimum of 1001 satoshis or more.
2. Be sure to include account_id=0 in the arguments of the send_to function.
ex: 
w.transactions_update()
tx_fee = 250
amount = 1750
receiver_address = 'receiver_address '
t = w.send_to(receiver_address, amount, account_id=0, fee=tx_fee, min_confirms=0, offline=False)
w.info(detail=5)

**Comment by mccwdev** [comment]
Just some remarks:
- The dust amount is currently set at 1000 satoshi, you can change this setting in the  network configuration file at networks.json
- The default value for account_id is 0, I don't think you have to specify it when you use only one account

**Reply by kernaldpark** [reply]
Thank you very much!
